.parcelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "@parcel/config-default"
+  ],
+  "optimizers": {
+    "*.{mjs,js}": [
+      "./scripts/fix-use-client.js"
```

Comment:
this is an optimizer instead of a babel plugin because it needs to run **after** parcel has bundled the code.